code/modules/admin/verbs/massmodvar.dm

```diff
@@ -453,6 +453,38 @@
 						if (A.type == O.type)
 							A.vars[variable] = O.vars[variable]
 
+	if(method)
+		if(istype(O,/mob))
+			for(var/mob/M in mob_list)
+				if(istype(M,O.type))
+					M.on_varedit(variable)
+
+		else if(istype(O,/obj))
+			for(var/obj/A in world)
+				if(istype(A,O.type))
+					A.on_varedit(variable)
+
+		else if(istype(O,/turf))
+			for(var/turf/A in world)
+				if(istype(A,O.type))
+					A.on_varedit(variable)

```

Comment:
Doesn't block only return a list of turfs?
If so, you could optimize that further like this:
`for(var/A in block(locate(1,1,1),locate(world.maxx,world.maxy,world.maxz)))`
That way you don't have the redundant type check when getting the list.